src/locale/locales/pt-BR/messages.po

```diff
@@ -2800,27 +2800,27 @@ msgstr ""
 
 #: src/components/KnownFollowers.tsx:169
 #~ msgid "Followed by"
-#~ msgstr ""
+#~ msgstr "Seguido por"
 
 #: src/view/com/profile/ProfileCard.tsx:190
 #~ msgid "Followed by {0}"
 #~ msgstr "Seguido por {0}"
 
 #: src/components/KnownFollowers.tsx:231
 msgid "Followed by <0>{0}</0>"
-msgstr ""
+msgstr "Seguido por <0>{0}</0>"
 
 #: src/components/KnownFollowers.tsx:217
 msgid "Followed by <0>{0}</0> and {1, plural, one {# other} other {# others}}"
-msgstr ""
+msgstr "Seguido por <0>{0}</0> e {1, plural, one {# other} other {# others}}"
 
 #: src/components/KnownFollowers.tsx:204
 msgid "Followed by <0>{0}</0> and <1>{1}</1>"
-msgstr ""
+msgstr "Seguido por <0>{0}</0> e <1>{1}</1>"
 
 #: src/components/KnownFollowers.tsx:186
 msgid "Followed by <0>{0}</0>, <1>{1}</1>, and {2, plural, one {# other} other {# others}}"
-msgstr ""
+msgstr "Seguido por <0>{0}</0>, <1>{1}</1>, e {2, plural, one {# other} outro {# others}}"
```

Comment:
shouldn't "one" be translated too?

Reply:
Do not translate inside {}

Reply:
so should we remove "outro" since it is also inside {}?

Reply:
Yes in line 2823 
`msgstr "Seguido por <0>{0}</0>, <1>{1}</1>, e {2, plural, one {# other} other {# others}}"`

Reply:
I think you want to translate `{# other}` and `{# others}`, as those are extra suffixes depending on how many other people are following the user, as used here:
<img width="409" alt="image" src="https://github.com/user-attachments/assets/d5904b88-cef7-49ba-90a8-8bb1a217d5aa">

Agree you don'tr translate `plural, one` or the `} other {` (i.e. things inside the _first_ `{}`).

Reply:
ok